### PR TITLE
Add preference to exclude ghosts from team scores

### PR DESCRIPTION
**Pull request by tienne-B**
As pointed out recently, the WUDC and EUDC rules show that team scores must not include scores from iron-men (except for their highest). This is 34.9.3.2 in WUDC. The current behaviour was to keep ghost scores, and is expected now. As such, a new preference is added to toggle between these calculations, but is set correctly in the preset.

### CODE REVIEW

File: tabbycat/results/result.py [hunk 1]
```diff
@@ -944,13 +946,18 @@ def teamscorebyadj_field_margin(self, adj, side):
     def teamscorebyadj_field_score(self, adj, side):
         return self.scoresheets[adj].get_total(side)
 
+    def _teamscore_score_component(self, adj, side):
+        if self.tournament.pref('teamscore_includes_ghosts'):
```
Comment (czlee) [inline tabbycat/results/result.py:950]: Australs results seem flipped, is this line missing a `not`?

**Review by czlee — COMMENTED** [review]
It's a bit weird now that the totalling is done in `Scoresheet` normally, but done in `ConsensusDebateResultWithScores` or `DebateResultByAdjudicatorWithScores` if ghosts are excluded. But I thought about it for quite a while and I don't have an obviously better solution 🤷🏻‍♂️

Looks good, just check up on the reversal in line 950?

File: tabbycat/results/result.py (hunk 1 above)
Comment (tienne-B) [reply tabbycat/results/result.py:950]: Fixed. Prefer avoiding a "if not, else" construction, so swapped the returns.

**Comment by czlee** [comment]
We should probably add unit tests for this kind of thing, lol

**Pull request merged**